### ISSUE TITLE
German 12.0 Segment missing train dev test TSV files

### ISSUE THREAD

**Issue by LozramA** [common-voice/cv-dataset#20]
The newest german segment "cv-corpus-12.0-delta-2022-12-07-de.tar" does not include the train.tsv dev.tsv and test.tsv.

**Comment by HarikalarKutusu** [comment]
Hey @LozramA, I don't know your workflow, but, if you have validated.tsv file, you could actually merge it with the v11.0 validated.tsv and use [CorporaCreator](https://github.com/common-voice/CorporaCreator) to generate a new train/dev/test set.

**Comment by LozramA** [comment]
On german V11 segment are missing all tsv files. so this is completely unusable.
I used CV12 full but thats getting now all to big for privatly available computer/GPU power. Was running many days to train and not really many epochs

**Comment by HarikalarKutusu** [comment]
Yes, if you not already have v11 in full you need to download it, unfortunately... And I know, it is a painful process.
